styles.scss

```diff
@@ -0,0 +1,230 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+
+//-------------------------------- Global styles
+body {
+  font-family: 'Poppins';
+
+
+.flex {
+    display: flex;
+}
+
+//-------------------------------- Start Upper Heading  
+    header > div {
```

Comment:
Im not sure how this looks on your computer but on mine it looks like this:
body .flex {
  display: flex;
}
body header > div {
  padding: 0 25px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  border-bottom: 1px solid lightgray;
}
body nav {
  width: 600px;
}

Where you basically add the body in front of every tag. This is not necessary as it is normally implied that the classes, id's and tags are already within the body. So it'll be better to write it like this:

.flex {
  display: flex;
}
header > div {
  padding: 0 25px;
  display: flex;
  justify-content: space-between;
  align-items: center;
  border-bottom: 1px solid lightgray;
}
nav {
  width: 600px;
}